modules/builtin/src/content-types/card.js

```diff
@@ -85,48 +86,49 @@ module.exports = {
           ]
         },
         type: 'group',
-        key: 'items',
-        renderType: 'buttons',
+        key: 'actions',
         label: 'fields::title',
         fields: [
           {
             type: 'text',
             key: 'title',
+            translated: true,
             label: 'module.builtin.types.actionButton.textLabel',
             placeholder: 'module.builtin.types.actionButton.textPlaceholder'
           },
           {
             type: 'select',
-            defaultValue: 'say',
+            defaultValue: 'Say something',
```

Comment:
The previous library (react-json-schema) used the litteral text as the ID of the selected action... So we would need to make a migration, but since it's already big enough, I left it there for now. But yeah definitely that should be changed in the future

Reply:
todo comment :) ?

Reply:
It's already in the SDK. 
![image](https://user-images.githubusercontent.com/42552874/87982158-55229b00-caa4-11ea-9741-eee629c7353a.png)